NOTICE.txt

```diff
@@ -12,6 +12,7 @@ ods-flutter/app/assets/ic_about_info.svg
 ods-flutter/app/assets/ic_components_atom.svg
 ods-flutter/app/assets/ic_guidelines_dna.svg
 ods-flutter/app/assets/ic_modules_molecule.svg
+ods-flutter/app/assets/il_buttons.svg
```

Comment:
Update Notice.txt (add this asset in list)

Reply:
sorry can you specify what is expected?

Reply:
No sorry, it's ok

Reply:
ok